packages/rooks/src/__tests__/useDebouncedValue.spec.ts

```diff
@@ -100,4 +110,42 @@ describe("useDebouncedValue", () => {
     });
     expect(result.current[0]).toBe(mockValue);
   });
+
+  describe("types", () => {
+    it("should return a union with null when initializeWithNull is true", () => {
+      doNotExecute(() => {
```

Comment:
Holy! I love this pattern of just testing types without running code. Very nice!